dbms/src/Storages/DeltaMerge/ReadThread/UnorderedInputStream.h

```diff
@@ -79,7 +79,12 @@ class UnorderedInputStream : public IProfilingBlockInputStream
         {
             return {};
         }
-        addReadTaskPoolToScheduler();
+        Stopwatch watch_read;
+        SCOPE_EXIT({
+            auto elapsed = watch_read.elapsedSeconds();
```

Comment:
Is it better or not that we use seperated metrics in DMSegmentThreadInputStream and UnorderedInputStream?

Reply:
Yes, seperate them is better

Reply:
fixed